docker-compose.yml

```diff
@@ -1,27 +1,80 @@
-version: '3'
+version: "3"
 
 services:
-  slurm:
-    build: .
-    hostname: slurmctl
-    stdin_open: true
-    tty: true
-    ports:
-      - "6818:6818"
-      - "6819:6819"
-      - "6820:6820"
-    cap_add:
-      - SYS_PTRACE
-      - SYS_ADMIN
-      - MKNOD
-      - SYS_NICE
-      - SYS_RESOURCE
-    security_opt:
-      - seccomp:unconfined
-      - apparmor:unconfined
-    volumes:
-      - ./volumes/lib:/var/lib/slurmd
-      - ./volumes/spool:/var/spool/slurm
-      - ./volumes/log:/var/log/slurm
-      - ./volumes/db:/var/lib/mysql
-      - ./volumes/jwt:/etc/slurm/jwt
+    slurmctl:
+        build: .
+        hostname: slurmctl
+        command: "controller"
```

Comment:
Doesn't look like this command does anything, it's the environment variable that's used. Is that right?

Reply:
Correct, yeah. This is from a previous iteration where I used the command rather than an env. But this meant you had to hardcode `exec bash` rather than `exec $@` - so went for env.

Pushed the fix now